results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/bf3bf13ab40c3157080a7ab344c831b9ad18b5eb/OnlineStoreReviewSentimentClassification.json

```diff
@@ -0,0 +1,62 @@
+{
+  "dataset_revision": "fb63ba1255f57054d411fe02bb5cec25cd6b150c",
+  "evaluation_time": 207.44641995429993,
+  "kg_co2_emissions": null,
+  "mteb_version": "1.12.10",
+  "scores": {
+    "train": [
+      {
+        "accuracy": 0.3216796875,
```

Comment:
How are the scores compared to before the change?

Reply:
[results/sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2/OnlineStoreReviewSentimentClassification.json ](https://github.com/embeddings-benchmark/mteb/pull/710/files#diff98ff58b6a6e6992951867b8eaea39479f276d45b2557200fdfa705609153f226)

before
`"accuracy": 0.319775390625,`
Now
`"accuracy": 0.3216796875,`

[results/intfloat__multilingual-e5-small/OnlineStoreReviewSentimentClassification.json](https://github.com/embeddings-benchmark/mteb/pull/710/files#diff-d1fe1e85259ae8e11833ae7dfe4aeafa2fa3b6556bac357c31ad3de849971ce7)
before
`"accuracy": 0.351953125,`
Now
`"accuracy": 0.362255859375,`